e2e/e2e_unreliable_network_test.go

```diff
@@ -0,0 +1,82 @@
+package e2e
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func TestE2E_Unreliable_Network(t *testing.T) {
```

Comment:
Is this a Fuzz or E2E test?

Reply:
It's fuzz test, will make adjustments as for previous fuzz test.

Reply:
Move to fuzz_xxx_test.go and make it a fuzz test.

Reply:
Fuzz tests moved to separate fail with appropriate test function name.